files/eruda/update.json

```diff
@@ -3,7 +3,6 @@
   "name": "eruda",
   "repo": "liriliri/eruda",
   "files": {
-     "basePath": "dist/",
```

Comment:
Why? The current version has the dist/ folder
https://github.com/liriliri/eruda/tree/v1.0.0/dist